test/integration/openssl.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Hmm, I think I still need this file for other scenarios. (Tests not run via ansible-test).

Like:
  * vyos: https://github.com/ansible/ansible/blob/devel/test/integration/vyos.yaml
  * ovs: https://github.com/ansible/ansible/blob/devel/test/integration/ovs.yaml